projects/gapps-html-to-google-doc/src/HtmlToGoogleDoc.js

```
@@ -0,0 +1,99 @@
+    this.processElement(root, body);
+  }
+
+  ensureWellFormedHtml(htmlContent) {
+    return `<root>${htmlContent}</root>`;
+  }
+
+  processElement(element, parent) {
+    const tagName = element.getName().toLowerCase();
+    switch (tagName) {
+      case 'h1':
+        const header1 = parent.appendParagraph(element.getText());
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)